packages/turf-concave/index.js

```diff
@@ -15,10 +15,9 @@ var distance = require('@turf/distance');
  * @param {FeatureCollection<Point>} points input points
  * @param {number} maxEdge the size of an edge necessary for part of the
  * hull to become concave (in miles)
- * @param {string} units used for maxEdge distance (miles or kilometers)
+ * @param {String} [units=kilometers] can be degrees, radians, miles, or kilometers
```

Comment:
Ops.. I just simply copied an example from TurfJS, just realized that `{string}` or `{String}` is not very consistent overall. I'll go fix the other JSDocs `units` that have `{String}`.

I normally put `{string}` as param, it's also explained in JSDocs
http://usejsdoc.org/tags-param.html